spec/features/can_add_and_edit_item_to_cart_as_guest_spec.rb

```diff
@@ -225,4 +224,27 @@
     expect(page).to have_link("View More Issues")
     expect(page).to have_link("View More Candidates")
   end
+
+  scenario "as guest to receive a flash notice and link after removing a donation" do
+    candidate = Candidate.create(name: "Ted Cruz", party: "Republican", bio: "Kim Davis")
+    issue = Issue.create(topic: "Gun Control", description: "Guns Guns Guns!")
+    candidate_issue = CandidateIssue.create(candidate: candidate, issue: issue, stance: "Give them the guns!")
+
+    visit issue_path(issue)
+    within("##{candidate_issue.id}") do
+      fill_in "Amount", with: 20
+      click_button "Donate"
+    end
+
+    visit cart_path
+
+    within("#donation-table") do
+      first(:link, 'Remove').click
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.